samples/basic/blinky/sample.yaml

```diff
@@ -10,3 +10,30 @@ tests:
     harness: led
     integration_platforms:
       - frdm_k64f
+
+  sample.basic.blinky.egpio_icmsg:
+    tags:
+      - LED
+      - gpio
+    filter: dt_enabled_alias_with_parent_compat("led0", "gpio-leds")
+    depends_on: gpio
+    harness: led
```

Comment:
There is no such harness `led`, it is dummy one.
Please change to `console` and add expected regex from sample.

Reply:
You can do sth similar to this:
https://github.com/zephyrproject-rtos/zephyr/pull/72730/files